modules/monitoring-granting-users-permission-to-configure-monitoring-for-user-defined-projects.adoc

```diff
@@ -16,11 +16,34 @@ You can grant users permission to configure monitoring for user-defined projects
 
 .Procedure
 
-* Assign the `user-workload-monitoring-config-edit` role to a user in the `openshift-user-workload-monitoring` project:
+. Assign the `user-workload-monitoring-config-edit` role to a user in the `openshift-user-workload-monitoring` project:
 +
 [source,terminal]
 ----
 $ oc -n openshift-user-workload-monitoring adm policy add-role-to-user \
   user-workload-monitoring-config-edit <user> \
   --role-namespace openshift-user-workload-monitoring
 ----
+
+. Verify that the user is correctly assigned to the `user-workload-monitoring-config-edit` role by displaying their associated role binding:
```

Comment:
The `their` may imply that each user gets their own role binding.
The same role binding can be used to bind all users to the `user-workload-monitoring-config-edit` role.
In the `oc` command above, because you didn't specify a role binding, a default one `user-workload-monitoring-config-edit` in this case, is used/created/updated.
(`oc adm policy add-role-to-user --help` for more details if you want)


```suggestion
. Verify that the user is correctly assigned to the `user-workload-monitoring-config-edit` role by displaying the associated role binding:
```
or `the related role`? or just `displaying the role binding` wdyt?

Reply:
Hello @machine424 thank you for the information! I did not know we could bind more users to same role binding, that is why I used the wording, so thank you for the clarification, I will keep that in mind and make the change as suggested :+1:

Reply:
Also because that implies that the rolebinding can have different name, I am adding it as a valuable and then adding an example command.